source/ncengine/physics/jolt/ShapeFactory.cpp

```diff
@@ -0,0 +1,115 @@
+#include "ShapeFactory.h"
+#include "Conversion.h"
+
+#include "ncengine/asset/AssetData.h"
+#include "ncjolt/ShapeUtility.h"
+
+#include "Jolt/Physics/Collision/Shape/BoxShape.h"
+#include "Jolt/Physics/Collision/Shape/CapsuleShape.h"
+#include "Jolt/Physics/Collision/Shape/ScaledShape.h"
+#include "Jolt/Physics/Collision/Shape/RotatedTranslatedShape.h"
+#include "Jolt/Physics/Collision/Shape/ScaledShape.h"
```

Comment:
These headers could do with a bit of a tidy. ScaledShape is duped and they could use some alphabetizing.